src/components/pill-selector-dropdown/__tests__/PillSelectorDropdown.test.js

```diff
@@ -1,115 +1,83 @@
 import React from 'react';
 import { List, Record } from 'immutable';
-import sinon from 'sinon';
```

Comment:
I refactored this file to remove `sinon` and to make use of a `getWrapper` function.